community.md

```diff
@@ -37,27 +32,55 @@ Short and brief description of the sort of content you will find on this page.
 
 <!-- Single Column Body Module -->
 
-<section id="connect-with-us-container" class="module">
-<aside class="module-aside table-of-contents" markdown="1">
-
-###### Contents
+<section id="who-is-part-of-the-community" class="module">
+<aside class="module-aside table-of-contents">
 
-1. [Connect with us](#connect-with-us-container 'Connect with us')
-2. [Become a contributor](#become-a-contributor-container 'Become a contributor')
+{%- include contents.html -%}
 
 </aside>
 <article class="module-content grid-x grid-padding-x">
 <div class="cell small-12" markdown="1">
 
-## Connect with us
+## Who is part of the community?
+
+Firefox extension developers are part of a vibrant, global community of creators who help millions of Firefox users around the world to customize their browsing experience. They’re ready to share their skills and expertise to help you develop your Firefox extension.
+
+</div>
+</article>
+</section>
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+<section id="connect-with-the-community" class="module">
```

Comment:
The `section` elements are a missing a heading here. A section should have a heading as direct child of section.

See https://www.w3.org/TR/html52/sections.html#the-section-element

> The section element represents a generic section of a document or application. A section, in this context, is a thematic grouping of content. Each section should be identified, typically by including a heading (h1-h6 element) as a child of the section element.

Reply:
Are you sure that a heading should be a direct child? My intent here was to have sections be flexible enough to contain an `article` and an `aside`, for instances when we have a related block of content in the right rail.

Reply:
Certainly 2 outline checkers I used can't see a heading for the section element including: https://validator.w3.org/nu/

<img width="1558" alt="Showing_results_for_ew_html_-_Nu_Html_Checker" src="https://user-images.githubusercontent.com/1514/60954225-8ba4d200-a2f6-11e9-98ce-37b26f5228c8.png">

Reply:
I see. I guess we would ideally have a heading inside the section, in addition to a heading in the nested article. In our case, that would be redundant. I like how the section creates a container for the article/aside siblings, but would you suggest replacing these elements with divs?